src/game.cpp

```diff
@@ -5582,6 +5582,29 @@ void Game::playerAnswerModalWindow(uint32_t playerId, uint32_t modalWindowId, ui
 	}
 }
 
+void Game::playerExecuteParsePacketEvent(uint32_t playerId, uint8_t recvbyte, NetworkMessage* message)
+{
+	Player* player = getPlayerByID(playerId);
+	if (!player) {
+		return;
+	}
+
+	if (!player->hasEventRegistered(CREATURE_EVENT_PARSE_PACKET)) {
+		return;
+	}
+
```

Comment:
From now everything creates a memory leak, only exception to this if creatureEvent->executeParsePacket is executed correctly and the networkmessage is added to the lua enviroment, then in this case it will be deleted by the gc.